EIPS/eip-1.md

```diff
@@ -232,6 +232,64 @@ Permitted Networking Specifications URLs must anchor to a specific commit, and s
 ^https://github.com/ethereum/devp2p/blob/[0-9a-f]{40}/.*$
 ```
 
+### Other Well Known Standards Bodies
+
+Links to other well known standards bodies specification may be included using normal markdown syntax. Be mindful of the requirements set for each SDO as these were chosen in order to maintain links to stable URLs.
+
+#### World Wide Web Consortium (W3C)
+
+Links to a W3C "Recommendation" status specification may be included using normal markdown syntax, where the content of the link is the title of the specification. For example, the following link would be allowed:
+
+```markdown
+[W3C Secure Context](https://www.w3.org/TR/secure-contexts/)
```

Comment:
EIP-5757 says that origins must "provide a method of uniquely identifying a particular revision of a resource". To that end, I'd recommend that we use "this version" links and not the latest version links.

Also, the title of that standard is "Secure Contexts", not "W3C Secure Contexts" as far as I could tell?

How about this for the example:

```suggestion
[Secure Contexts](https://www.w3.org/TR/2021/CRD-secure-contexts-20210918/)
```